templates/views/newshub.pug

```diff
@@ -19,8 +19,6 @@ html(lang='en')
 
     //- Customise the stylesheet for your site by editing /public/styles/site.sass
     link(href="/styles/css/site.min.css?version="+Date.now(), rel="stylesheet")
-    script(async, defer, data-domain="faforever.com", src="https://plausible.faforever.com/js/plausible.js")
-    script(src="https://kit.fontawesome.com/33e7e258d3.js" crossorigin='anonymous')
```

Comment:
plausible http 404
kit: http 403

don't know why this was loaded, but the newshub still looks okay

Reply:
Plausible is currently disabled on the server.